cernopendata/modules/fixtures/data/docs/cern-open-data-release-2024/cern-open-data-release-2024.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "author": "CERN",
+    "body": {
+      "content": "cern-open-data-release-2024.md",
+      "format": "md"
+    },
+    "collections": [
+      {
+        "primary": "News"
+      }
+    ],
+    "date_published": "2024-01-31",
+    "short_description": {
+      "content": "CERN Open Data release 2024"
+    },
+    "slug": "cern-open-data-release-2024",
```

Comment:
The SLUG says "cern-open-data-release-2024" and the title says "CERN Open Data Release 2024". This could be easily misunderstand as presenting an open data release rather than presenting a repository infrastructure upgrade. Moreover, there may be more repository updates coming later in the year 2024, so we should leave the room for those, and be more descriptive about this change. For example: "Upgrade of the CERN Open Data portal web interface" or "Upgrade of search functionality".

Basically, let's pick some leading news time (which could be Angular->React or the improved search), and let's model the title and the slug after that. In this way, the news item will be more end-user-facing. And the other "minor" news times would come along as a by-product without particular highlighting.